src/content/blog/2024-02-23-dnsmasq-custom-dns-resolvers-for-specific-domains/index.md

```diff
@@ -1,15 +1,14 @@
 ---
-title: "dnsmasq - Custom DNS resolvers for specific domains"
-seoTitle: "dnsmasq - Custom DNS resolvers for specific domains"
-slug: "dnsmasq-custom-dns-resolvers-for-specific-domains"
+title: 'dnsmasq - Custom DNS resolvers for specific domains'
+seoTitle: 'dnsmasq - Custom DNS resolvers for specific domains'
+slug: 'dnsmasq-custom-dns-resolvers-for-specific-domains'
 description: "It's possible to setup custom DNS resolvers for certain domains using a dnsmasq config file. Requests still pass through pihole, for ad-blocking capabilities."
```

Comment:
..I asked the other question because Prettier seems to have handled this intelligently!